### DISCUSSION TITLE
Web dev at Open Science Tools Ltd

### DISCUSSION THREAD

**Discussion by peircej** [pixijs/pixijs#7920] [Jobs]
At Open Science Tools we're currently looking for someone with a solid understanding of accelerated graphics in the browser (e.g. WebGL or Pixi), to join our team and lead development of PsychoJS, our open-source lib to run experiments in behavioural sciences (psychology, neuroscience, linguistics etc.).

It's a slightly unusual kind of web dev, where we're trying to optimise/measure timing to a high precision, and I thought some of the people on this list might be interested in that challenge!

If you want to know more about why we need high temporal precision see our paper at https://peerj.com/articles/9414/

To apply, check out https://opensciencetools.org/vacancies.html#job003

let me know if you have any queries
Jon

**Comment by ivanpopelyshev** [comment]
re-posted in russian webgl jobs telegram channel
